TokenManagementSystem/resources/views/auth/login.blade.php

```diff
@@ -2,19 +2,88 @@
 
 @section('content')
 
+<head>
```

Comment:
Instead of writing js validation script in blade file, keep it in abc.js file and cut paste and just use link of abc.js  file

Reply:
Also do the same for other files

Reply:
we were doing same for register.js  but was not working, we have just wriiten script on onchange event to remove unnecessary warnings

Reply:
Its not working for login so should I remove that?